### DISCUSSION TITLE
Get enum value

### DISCUSSION THREAD

**Discussion by jmpangilinan** [vfsfitvnm/frida-il2cpp-bridge#356] [Q&A] [answered]
```
enum RunType : System.Enum
{
    System.Int32 value__; // 0x10
    static RunType None = -1;
    static RunType Fast = 0;
    static RunType Slow = 1;
}
```

how do I get the value of Fast? which is equal to 0

`AssemblyCSharp.class('RunType').field<Il2Cpp.ValueType>("Fast").value`

but it just gives me "Fast" not 0, sorry for the dumb question.

Edit:

`AssemblyCSharp.class('RunType').field<Il2Cpp.ValueType>("Fast").toString().split(" = ")[1].replace(';','');`

this works but very ugly solution.

**Comment by vfsfitvnm** [comment, marked as answer]
Here you are:
```ts
AssemblyCSharp.class("RunType").field<Il2Cpp.ValueType>("Fast").value.field<number>("value__").value
```

**Reply by jmpangilinan** [reply]
Thanks a lot @vfsfitvnm
